ovsInit.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+#Start OVS in the Contiv container
+
+set -euo pipefail
+
+if [[ $(lsmod | cut -d" " -f1 | grep -q openvswitch) -eq 1 ]]; then
+	modprobe openvswitch
+fi
+
+mkdir -p /var/run/openvswitch
```

Comment:
both paths can be on one line:

`mkdir -p /var/run/openvswitch /var/log/contiv`